CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## 2.0.3 / 2018-06-09
 
+* Alter default host on devevopment mode [#1369](https://github.com/sinatra/sinatra/issues/1369) by David Silveira
```

Comment:
2.0.3 is already released, so this line is in the wrong place

Reply:
Thanks, I'll fix it.